### ISSUE TITLE
OIDC scope configuration key name mismatch

### ISSUE THREAD

**Issue by kjoconnor** [oxyno-zeta/s3-proxy#109]
**Describe the bug**
The documentation as well as the helm chart differ from the source code in regards to OIDC scope specification.

**To Reproduce**
Configure an OIDC provider:
```yaml
authProviders:
  oidc:
    provider1:
      scopes:
        - openid
        - email
        - profile
        - groups
```

Browse to s3-proxy unauthenticated and you will be redirected to the OIDC provider, but it will only have requested the default scopes of `[ openid, email, profile ]`, it will not have added `groups`.

**Expected behavior**
I would expect all of the scopes I requested be added to the auth request to the OIDC provider.

**Version and platform (please complete the following information):**
Running on k8s 1.19 on IBM Cloud, with the latest helm chart and s3-proxy image.

**Additional context**
It seems to just be a mismatch between the [documentation](https://github.com/oxyno-zeta/s3-proxy/blob/080af1418c63961f53ad11e438c3ea98302820ba/docs/configuration.md#oidcauthconfiguration) as well as [helm chart](https://github.com/oxyno-zeta/helm-charts/blob/080af1418c63961f53ad11e438c3ea98302820ba/stable/s3-proxy/values.yaml#L95) and the [code](https://github.com/oxyno-zeta/s3-proxy/blob/d92ca2c5dc6954da38254140c1fcf074137963ba/pkg/s3-proxy/config/config.go#L110). The docs and helm chart say to use `scopes`, but the code itself uses `scope`. Not sure which direction you prefer to change it.

**Comment by oxyno-zeta** [comment]
Hello @kjoconnor 

Thanks for your issue. You are right. It is clearly a bug. I will fix it now. 
It will be "scopes".

**Comment by oxyno-zeta** [comment]
A new release has been released with the fix: v3.0.4 . Should be ok now.

**Comment by kjoconnor** [comment]
Thank you for the quick fix!
